pkgs/development/tools/wasm-bindgen-cli/default.nix

```
@@ -0,0 +1,27 @@
+{ rustPlatform, fetchFromGitHub, lib, openssl, pkgconfig, ... }:
+
+rustPlatform.buildRustPackage rec {
+  pname = "wasm-bindgen-cli";
+  version = "0.2.50";
+
+  src = fetchFromGitHub {
+    owner = "rustwasm";
+    repo = "wasm-bindgen";
+    rev = version;
+    sha256 = "1axlp4qa2fi79giqwcq105qb9n0znfmknrmnqxy6vdzqzgfszqfq";
+  };
+
+  buildInputs = [ openssl ];
```

Fails to build on darwin:

```
"Security" "-lSystem" "-lresolv" "-lc" "-lm"
  = note: ld: framework not found Security
          clang-7: error: linker command failed with exit code 1 (use -v to see invocation)
error: aborting due to previous error
```

```suggestion
  buildInputs = [ openssl ] ++ stdenv.lib.optionals stdenv.isDarwin [ darwin.apple_sdk.frameworks.Security ];
```